lib/archethic_web/graphql_schema/resolver.ex

```diff
@@ -122,13 +122,19 @@ defmodule ArchethicWeb.GraphQLSchema.Resolver do
     end)
   end
 
-  def get_inputs(address) do
-    case Archethic.get_transaction_inputs(address) do
-      {:ok, inputs} ->
-        {:ok, Enum.map(inputs, &TransactionInput.to_map/1)}
-
-      {:error, _} = e ->
-        e
+  def get_inputs(address, paging_offset \\ 0, limit \\ 0) do
+    inputs =
+      address
+      |> Archethic.get_transaction_inputs(paging_offset)
+      |> Enum.map(&TransactionInput.to_map/1)
+      |> Enum.sort_by(& &1.timestamp, {:desc, DateTime})
```

Comment:
Sorting by timestamp is a bit confusing there I think, or it should be the same sorting in the GettransationInputs message. Here the problem:
I request all transaction inputs and I get 
[1, 2 ,3 , 4, 5] (1 is the more recent)
Then I want to get inputs from offset 2, I expected to have [3, 4, 5]
but I get [1, 2, 3] because offset in the message is based on an asc sort and not desc sort